test/NNODE2_tests2.jl

```diff
@@ -0,0 +1,40 @@
+#using Plots
+using Test, Flux
+using DiffEqDevTools
+using DiffEqBase, NeuralNetDiffEq
+
+function f(ddu, du, u, p, t)
+	ddu[1] = (-p[3]*u[1]-p[4]*(u[1]-u[2]))/p[1]
+	ddu[2] = (-p[4]*(u[2]-u[1]))/p[2]
+end
+function f(du, u, p, t)
+	[(-p[3]*u[1]-p[4]*(u[1]-u[2]))/p[1], (-p[4]*(u[2]-u[1]))/p[2]]
+end
+p = [1.0f0, 1.0f0, 6.0f0, 4.0f0]
+u0 = [1.0f0, 0.0f0]
+du0 = [2.0f0, 0.0f0]
+prob = SecondOrderODEProblem{false}(f, [u0], [du0], (0.0f0, 3.0f0), p)
```

Comment:
why do you use `[u0], [du0]`? Probably, it will be easy just to use `u0, du0` and change accordingly `phi`

Reply:
It's actually because if I don't, it just gets passed as a (4,) and i can't easily separate u0 and du0 when calling `u0, du0 = prob.u0` in solver